cmd/commands/server.go

```diff
@@ -65,6 +69,8 @@ func NewServerCommand() *cobra.Command {
 	command.Flags().StringVar(&managedNamespace, "managed-namespace", sharedutil.LookupEnvStringOr("NAMESPACE", "numaflow-system"), "The namespace that the server watches when \"--namespaced\" is \"true\".")
 	command.Flags().StringVar(&baseHref, "base-href", "/", "Base href for Numaflow server, defaults to '/'.")
 	command.Flags().BoolVar(&disableAuth, "disable-auth", false, "Whether to disable authentication and authorization, defaults to false.")
-	command.Flags().StringVar(&dexServerAddr, "dex-server-addr", "http://numaflow-dex-server:5556", "The address of the Dex server.")
+	command.Flags().StringVar(&dexServerAddr, "dex-server-addr", "http://numaflow-dex-server:5556/dex", "The actual address of the Dex server for the reverse proxy to target.")
+	command.Flags().StringVar(&dexProxyAddr, "dex-proxy-addr", "https://localhost:8443/dex", "The proxy address of the Dex server.")
```

Comment:
Can we get the proxy address from the server-addr?

Reply:
If serverAddr is localhost then proxy is also localhost... otherwise proxy will be `https://numaflow-server:8443/dex`